recipes-ni/nisdbootconfig/files/nisdbootconfig

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# check if target is NOT using ubifs (mtd)
+if ! cat /proc/mtd | grep -qs mtd ; then
+
+    # configure the partitions to mount by label in fstab
+    sed -i '/ubi/d' /etc/fstab
+    echo LABEL=nibootfs         /boot                ext4       sync           0  0 >>/etc/fstab
+    echo LABEL=niconfig         /etc/natinst/share   ext4       sync           0  0 >>/etc/fstab
```

Comment:
I assume the intention here is to remove entries from `/etc/fstab` which are of the type `ubi` and replace them with the `ext4` variants. Could you add some whitespace characters to the regex match to make it more precise.

eg. `sed -i '/\W\+ubi\W\+/d' /etc/fstab`

Reply:
I tried /\W\+ubi\W\+/d and it did not work as expected. The original string looks like this:
ubi0:bootfs          /boot                ubifs      noatime,sync          0  0

I'll change it to '/ubifs/d' instead to be more specific

Reply:
Well, at the least, you need to escape the `+` control character. But it's also possible that the busybox version of `sed` we have installed doesn't interpret the string the same way (or support the `\W` metacharacter).

I guess I'm fine with changing the match to `ubifs`, so long as that is reasonably unlikely to occur in the fstab in any other place.